lumicks/pylake/__init__.py

```diff
@@ -2,7 +2,10 @@
                         __title__, __url__, __version__)
 
 from .file import *
+from .fitting.models import *
 from .correlated_stack import CorrelatedStack
+from .fitting.fit import FdFit
+from .fitting.detail.parameter_trace import parameter_trace
```

Comment:
This probably doesn't belong in `detail` if it's being made available as public API anyway.

Reply:
Ah yes. Originally, I was planning to make it accessible through `FdFit`, but I think a free function is better now. I'll move it!